libs/api/books/src/lib/reading-list.service.ts

```diff
@@ -28,4 +28,18 @@ export class ReadingListService {
       return list.filter(x => x.bookId !== id);
     });
   }
+
+  async markBookAsFinished(id: string): Promise<ReadingListItem> {
```

Comment:
You've marked this as `async` but you're not actually returning a Promise.

Reply:
I made the appropriate change